@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -185,7 +185,7 @@ export interface BaseTechniqueParams {
      * If not specified in style file, [[StyleSetEvaluator]] will assign monotonically increasing
      * values according to style position in file.
      */
-    renderOrder: number;
+    renderOrder?: DynamicProperty<number>;
```

Comment:
Slighly related.
I would remove `DynamicProperty` from these definitions and allow everything to be dynamic property - that's what we support though.
But that's another patch i guess.

Reply:
yes, i agree that's coming with an upcoming patch.